### ISSUE TITLE
Plot of energymap and roseplot

### ISSUE THREAD

**Issue by JorickVel** [RubendeBruin/WaveDave#6]
A. When printing the roseplot to a pdf the figure is not included correctly, see below. 

B. Further I would like to be able to set the vertical axis of the energymap and roseplot (e.g. [0 15] sec). 

The EnergySection() objects includes an input cmap. Need some help to make it work (example does not work for me, guess Im doing something wrong). 

Good
![afbeelding](https://github.com/RubendeBruin/WaveDave/assets/166730208/3e719193-205f-401f-9819-9da3b8bc3df6)

Not good 
![afbeelding](https://github.com/RubendeBruin/WaveDave/assets/166730208/4413e8f3-80d3-484f-a591-45ccb293fb77)

**Comment by RubendeBruin** [comment]
Can not reproduce, can you provide a minimum reproducable example?

**Comment by JorickVel** [comment]
alstu

[debug roseplot in pdf.txt](https://github.com/RubendeBruin/WaveDave/files/14960355/debug.roseplot.in.pdf.txt)

**Comment by RubendeBruin** [comment]
Dat geeft mij dit:
![image](https://github.com/RubendeBruin/WaveDave/assets/34062862/151fce56-0d41-4b8a-bf8a-6cf6027d071a)

maar misschien is mijn "now" anders?

**Comment by RubendeBruin** [comment]
I've plotted all of them and none of 'm show up purple:

```pyhton
Settings.DATE_FORMAT = "%a %d %b\n%H:%m"

# read 2D buoy data
buoy2D = Spectra.from_obscape(
    r"C:\00_Offshore Decision Support\Obscape\Data L7\spec2D",
    datetime.now() - timedelta(days=2.5),
    datetime.now(),
)

# plot 'm all
for i in range(buoy2D.number_of_spectra):
    buoy2D.plot_spectrum_2d(i)
    plt.tight_layout()
plt.show()
```

**Comment by RubendeBruin** [comment]
Moved issue (b) to separate isse #9

**Comment by RubendeBruin** [comment]
It seems to be a fpdf rendering issue introduced with upgrading to fpdf2-2.7.8

Link: (link)[https://github.com/py-pdf/fpdf2/issues/1147]

Can be solved by downgrading to fpdf 2.7.6

`pip install fpdf2=2.7.6`
